### PR TITLE
Create Chinese CFP

### PR DESCRIPTION
**Pull request by kshyatt**
Many thanks to @GiggleLiu for this.

### CODE REVIEW

**Comment by GiggleLiu** [comment]
@johnnychen94 Do you have time to help us review briefly about this notice? Briefly going through it would be fine, in case I made some naive mistakes

File: 2022/cfp_zh.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
+# JuliaCon 2022 报告和讨论提案征集
+这则报告征集通知于 2021-04-01 11:59 (UTC) 过期。
```
Comment (GiggleLiu) [inline 2022/cfp_zh.md:2]: We need to change the date once the English version is fixed.

File: 2022/cfp_zh.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+- 普通报告、快速报告、海报和小型交流会在一起评审，分为两轮。在第一轮中，所有的评审人都会被安排评审 X 个提案，每个提案得到 3 次评审。每个评论都应该包括一个分数和一个评论，其中至少有 1 句话用于概述他们对提案的想法。如果有任何评审人退出，则把提案重新分配给活跃评审人。在这一轮之后，委员会将确保每个提案都经过 3 次审查。
+
+- 两轮评审后是遴选阶段。分数最高一档的提案会被直接接受，最低一档的提案将被拒绝或重新分类为中等提案。然后将标记中等的所有提案再额外审核。第 1 阶段选择要考虑的“顶级提案”的数量取决于报告人的名额与收到的提案的数量，并且通常此类报告有 4+ 的平均分数而且没有任何负面的评论。
+
+- 在第二次遴选中，评审委员会将根据评审人的评分、意见、重新分类等情况，决定最终是否录用。被拒绝的提案将在最终决议之前讨论是否重新分类。
+
+- 在对所有提案进行分类后，将发布接收或拒绝的结果。
+
+- 将暂时保留关于 Google Summer of Code (GSoC) 学生快速报告的特别座谈会，并最终由审查委员会直接审查。这是因为这些提案的批准或拒绝是在 GSoC 学生被接受之前。
```
Comment (johnnychen94) [inline 2022/cfp_zh.md:156]:
```suggestion
- 由于 Google Summer of Code (GSoC) 学生的宣布发生在会议提案的评审之后，因此针对 Google Summer of Code (GSoC) 学生的快速报告的评审将延后进行并由评审委员会直接处理。
```

**Review by johnnychen94 — COMMENTED** [review]
Free free to accept or ignore the suggestions I made.

**Comment by GiggleLiu** [comment]
Thank you johnney, I really like the idea of adding english title along side the chinese version to avoid confusion.
I have review all your comments and I think all the changes (except a typo) make sense and will make the statements more clear. Cheers!

File: 2022/cfp_zh.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+研讨会一般是一个三小时的交互式的需要动手编程的环节，演示者在与会者面前实时编程并共享代码，以便与会者跟进。研讨会将在正式会议开始之前举行。此环节的主题可以是关于某个软件包的教程或关于某个领域的教程等。*注意：* 由于这是 JuliaCon 第一年接受非英语研讨会的提案，如果您想提议非英语研讨会，请尽量降低研讨会的内容门槛。
+
+### 虚拟海报
+由于今年 JuliaCon 2022 仅有在线的会议，海报展示者不是准备实体海报，而是提供一个关于他们提案的主题的 3 分钟视频。这段视频将在专用聊天室中播放，加入房间的与会者可以快速观看视频并加入有关内容的讨论。
+
+### 兴趣小组
+兴趣小组会议是围绕特定主题进行的有组织的讨论。例如，软件包维护者可以利用这些时间组织会议并招募贡献者，或者可以让社区成员为 Julia 教材或招聘多样化的人才而组织讨论。这些会话将持续一小时，并且不会录制视频。
+
+### 使用体验
+这个环节旨在迎接社区中的新成员。主题可以包括您在 Julia 中最喜欢的功能、Julia 如何帮助您解决某个问题、您在大学课堂上使用 Julia 解决的一个有趣的问题，或者更一般地，Julia 如何成为您工作流程的一个很好的补充。这些报告每个持续三分钟，将会穿插在会议的不同环节中。此环节没有回答问题的部分，如果您希望大家线下向您提问，我们鼓励您在幻灯片上显示您的联系信息，例如 微信二维码，twitter 账号， github 账号或电子邮件。我们特别欢迎 Julia 的新人利用这个环节与社区分享他们的感受。如果您是在教学中使用 Julia 的教师或教授，我们鼓励您邀请您的学生申请参与这个环节并与 Julia 社区分享他们的经验。
```
Comment (tk3369) [inline 2022/cfp_zh.md:55]:
```suggestion
这个环节旨在迎接社区中的新成员。主题可以包括您在 Julia 中最喜欢的功能、Julia 如何帮助您解决某个问题、您在大学课堂上使用 Julia 解决的一个有趣的问题，或者更一般地，Julia 如何成为您工作流程的一个很好的补充。这些报告每个持续三分钟，将会穿插在会议的不同环节中。此环节没有回答问题的部分，如果您希望大家线下向您提问，我们鼓励您在幻灯片上显示您的联系信息，例如 微信二维码，Slack 账号, Humans-of-Julia Discord 账号, Zulip 账号, Twitter 账号， Github 账号或电子邮件。我们特别欢迎 Julia 的新人利用这个环节与社区分享他们的感受。如果您是在教学中使用 Julia 的教师或教授，我们鼓励您邀请您的学生申请参与这个环节，并与 Julia 社区分享他们的经验。
```

File: 2022/cfp_zh.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+提交提案（摘要）时，您可以选择多个类别。
+
+
+# 提案提交流程
+为避免与报告者相关的选择性偏见，提案用匿名的方式提交。由于执行双盲是不可能的，因为大多数提交都会链接到开源的代码，我们只能尽力确保对提交的公正评审。如果您要提交非英语的提案，您可以使用您选择的语言提交。此外，请提供您的提案的英文版本（允许机器翻译！）。我们会努力找到能流利使用您所选语言的人士来审核您的提案。但如果没有找到，我们将使用它的英文翻译以便将其提交给更广泛的审稿人。如果您选择使用与英语不同的语言进行汇报，请在摘要中注明这一点（例如，“本次研讨会将以泰语进行。”或“我将用捷克语进行 Julia 使用经验的报告。”）。
+
+如果您要提交您的研讨会或兴趣小组对时间有特别的要求，请在您的提案中注明这一点。例如，如果您想用普通话举办研讨会，并希望在亚洲参加者方便的时间举行，请在您的摘要中注明。
+
+### 视频录制和材料
+JuliaCon 2022 将是一个在线会议，为了方便会议的安排，我们要求我们的报告人在会议前提交他们报告的视频版本。我们会在会议结束后公开所有视频材料。如果出于任何原因，您希望不公开您的视频，请通过 juliacon@julialang.org 与我们联系。最后，我们还会要求您在 Creative Commons（默认：禁止商业用途）或其他开源许可下提供您的材料和视频。如果您想用英语和另一种语言进行汇报，我们会要求您提交两个视频版本（每种语言各一个）。
```
Comment (tk3369) [inline 2022/cfp_zh.md:66]: Is Creative Commons the default license? Or else we should reference a different license here.

File: 2022/cfp_zh.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+为了让更多人关注，我们将要求您在提交的材料中包含相关翻译。请密切关注您的电子邮件，以了解有关提交过程的详细信息。
+
+### 会议论文集
+今年，我们将会为有意愿的参与者提供机会把他们的工作作为 JuliaCon 会议论文集的一部分发表。更多细节即将发布。
+
+### 报告人的注册
+我们希望我们的报告人能够注册参加会议。由于 JuliaCon 2022 是一个完全在线的会议，注册将是免费的。
+
+### 如何联系我们
+您可以通过 juliacon@julialang.org 向我们提出问题和疑惑。
```
Comment (tk3369) [inline 2022/cfp_zh.md:79]:
```suggestion
您可以通过 juliacon@julialang.org 向我们提出问题。
```

File: 2022/cfp_zh.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+
+
+# 附录
+## 评审指南和流程
+作为参考，以下是评审人在评审您提交的提案时使用的指南和流程。
+
+评审人的作用是确保 JuliaCon 上呈现的内容的质量。
+
+## 利益冲突
+在存在利益冲突的情况下，评审人承诺退出评审并通知组织者以找到其他评审人。任何评审人都不应该对身为作者之一或存在其他形式利益冲突的任何报告进行评审。有关定义和示例请参阅 [PNAS 指南](https://www.pnas.org/page/authors/conflict-of-interest)。利益冲包括与工作或者其作者存在的任何与财务或者知识产权上的关联（参考上面的 PNAS 指南）。
```
Comment (tk3369) [inline 2022/cfp_zh.md:93]:
```suggestion
在存在利益冲突的情况下，评审人承诺退出评审并通知组织者以找到其他评审人。任何评审人都不应该对身为作者之一或存在其他形式利益冲突的任何报告进行评审。有关定义和示例请参阅 [PNAS 指南](https://www.pnas.org/page/authors/conflict-of-interest)。利益冲突包括与工作或者其作者存在的任何与财务或者知识产权上的关联（参考上面的 PNAS 指南）。
```

File: 2022/cfp_zh.md (hunk 4 above)
Comment (GiggleLiu) [reply 2022/cfp_zh.md:66]: It is directly translated from here: https://github.com/JuliaCon/www.juliacon.org/blob/044465e2e2360f32d79622da7459d059d2620d2a/2022/cfp.md?plain=1#L63

**Review by ErikQQY — COMMENTED** [review]
Some suggestions :)

**Pull request merged**